katello-selinux-enable

```diff
@@ -11,8 +11,13 @@ do
     /usr/sbin/semanage module -S $selinuxvariant \
       -a /usr/share/selinux/${selinuxvariant}/katello.pp.bz2
 
-    /usr/sbin/semanage port -E | grep -q elasticsearch_port_t || \
-      echo "port -a -t elasticsearch_port_t -p tcp 9200-9300" > $TMP
+    # Previously elasticsearch_port_t was defined in Foreman - move it
+    /usr/sbin/semanage port -E | grep -qE '\s+elasticsearch_port_t\s+' && \
+      semanage port -m -t katello_elasticsearch_port_t -p tcp 9200-9300
```

Comment:
lzap explained on IRC that semanage will apparently move the port definitions from elasticsearch_port_t to katello_elasticsearch_port_t when running this command.  I'm surprised, but haven't tested it.